Arrowgene.Ddon.GameServer/Handler/StageAreaChangeHandler.cs

```diff
@@ -15,9 +15,53 @@ public class StageAreaChangeHandler : StructurePacketHandler<GameClient, C2SStag
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(StageAreaChangeHandler));
 
         // List of "safe" areas, where the context reset NTC will be sent.
-        // TODO: Complete with all the safe areas. Maybe move it to DB or config?
+        // TODO: Complete with all the safe areas.
         private static readonly HashSet<uint> SafeStageIds = new HashSet<uint>(){
-            2 // White Dragon Temple
+            // Lobbies
+            2, // White Dragon Temple
+            3, // Audience Chamber
```

Comment:
I notice when looking at some packets, we can see these as well. The IsBase field gets set.

WDT
> {"StageNo": 2, "IsBase": true, "StageFeatureList": [{"Value": 1001}, {"Value": 2001}], "Unk0": [], "Unk1": [], "Error": 0, "Result": 0}}

Audience Chamber
> {"Structure": {"StageNo": 3, "IsBase": true, "StageFeatureList": [{"Value": 1001}, {"Value": 2001}], "Unk0": [], "Unk1": [], "Error": 0, "Result": 0}}

// Hallow of Beginnings: Gathering Area
> {"Structure": {"StageNo": 384, "IsBase": true, "StageFeatureList": [], "Unk0": [], "Unk1": [], "Error": 0, "Result": 0}}

// Glyndwr Centrum
> {"StageNo": 377, "IsBase": true, "StageFeatureList": [{"Value": 1001}, {"Value": 2008}], "Unk0": [], "Unk1": [], "Error": 0, "Result": 0}}

Reply:
Can we move this into some common area which it can be queried? In the quest branch, I made a `StageManager` class which would probably be a good place for this kind of information.

Reply:
Sounds like a good idea. If you're merging this into your quest branch then i'll close my PR